libs/data-services/src/lib/tickets.service.ts

```diff
@@ -16,6 +16,8 @@ export class TicketsService {
     const variables = {
       start: event ? event.pageIndex * event.pageSize: 0,
       limit:  event ? event.pageSize  : 10,
+      sort: 'employeeName:asc,employeeName:desc',
+      where: event ? {Name_contains: event.where} : {}
```

Comment:
likewise this is employees syntax. It's why i removed so we don't have to worry about now